src/current/cockroachcloud/cloud-api.md

```diff
@@ -335,12 +478,71 @@ If the request was successful, the API will return detailed information about th
 Where:
 
 - `{node_name}` is the name of the node.
-- `{region_name}` is the cloud infrastructure provider region where the cluster is located.
-- `{status}` is the status of the node. Possible values are: `LIVE` and `NOT_READY`.
+- `{region_name}` is the cloud provider region where the cluster is located.
+- `{status}` is the status of the node: `LIVE` or `NOT_READY`.
+
+## Set or update resource limits for a Basic cluster
+
+To specify the maximum RU or storage limits for a cluster, send a `PATCH` request to the `/v1/clusters/{cluster_id}` endpoint with an updated `serverless.usage_limits` field.
+
+{{site.data.alerts.callout_info}}
+The `spend_limit` field, which was deprecated in Serverless, is not supported on Basic or Standard. Instead, use `usage_limits`.
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_success}}
+The service account associated with the secret key must have the Cluster Administrator or Cluster Developer [role]({% link cockroachcloud/authorization.md %}#organization-user-roles).
+{{site.data.alerts.end}}
+
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="curl"><strong>curl</strong></button>
+    <button class="filter-button page-level" data-scope="raw"><strong>Raw</strong></button>
+</div>
 
-## Set the maximum resource limits of a Serverless cluster
+<section class="filter-content" markdown="1" data-scope="curl">
+
+{% include_cached copy-clipboard.html %}
```

Comment:
This is the right place to show usage limits in use, rather than above.

Reply:
👍